src/main/webapp/resources/js/pages/projects/associated-projects/AssociatedProjects.jsx

```diff
@@ -3,14 +3,13 @@
  */
 import React from "react";
 import ViewAssociatedProjects from "./ViewAssociatedProjects";
-import { getI18N } from "../../../utilities/i18n-utilities";
 import { TabPaneContent } from "../../../components/tabs";
 
 export default function AssociatedProjects() {
   return (
     <TabPaneContent
-      title={getI18N("AssociatedProjects.title")}
-      subTitle={getI18N("AssociatedProjects.subTitle")}
+      title={i18n("AssociatedProjects.title")}
```

Comment:
Doesn't look like this is properly internationalizing
![image](https://user-images.githubusercontent.com/8951708/70453584-a37fae00-1a6e-11ea-8d0d-fee05d4339ac.png)

Reply:
Fixed in ab2fb353054a2147fa51b70083e430075daf61e4